fearless/Common/Helpers/WeakWrapper.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+final class WeakWrapper {
+    weak var target: AnyObject?
+
+    init(target: AnyObject) {
+        self.target = target
+    }
+}
```

Comment:
I suggest using generics here

Reply:
Not an option because we should specify concrete type of the value in the dictionary. So, let items: [String: WeakWrapper<we need concrete type here but what should we put?>]